qa/test-db-rolling-update/test-old-engine/src/test/java/org/operaton/bpm/qa/rolling/update/callactivity/CompleteProcessWithCallActivityTest.java

```diff
@@ -38,7 +36,7 @@ public class CompleteProcessWithCallActivityTest extends AbstractRollingUpdateTe
 
   protected RuntimeService runtimeService;
 
-  @Before
+  @BeforeEach
```

Comment:
This is undesired! The issue is not about migrating from JUnit 4 to JUnit 5!
Please leave usages of.`org.junit.Before`, `org.junit.Test` etc. untounched! It is ONLY about the `Assert` class.